src/gradio-radio-station.vala

```diff
@@ -88,6 +89,18 @@ namespace Gradio{
 			set{_bitrate = value;}
 		}
 
+		public string techinfo {
+			get{
+				if(strcmp(_codec, "UNKNOWN")!=0){
+					_techinfo = _codec + "-" + _bitrate + "kB/s";
+				}else{
+					_techinfo = "???-" + _bitrate + "kB/s";
```

Comment:
The bitrate information is from the radio-browser.info database. Sometimes it is "0", which isn't true. As a workaround we should use "? kB/s" instead of "0 kB/s".

I'll contact the maintainer of the database.

Reply:
https://github.com/segler-alex/radiobrowser-api/issues/38

I think this is not going to be fixed soon... So we should use the workaround meanwhile.

Reply:
Okay, I've updated this info.